data/scripts.js

```diff
@@ -159,154 +167,150 @@ exports.BattleScripts = {
 
-		if (!sourceEffect || sourceEffect.id === 'pursuit') {
-			let extraPP = 0;
-			for (let i = 0; i < targets.length; i++) {
-				let ppDrop = this.singleEvent('DeductPP', targets[i].getAbility(), targets[i].abilityData, targets[i], pokemon, move);
-				if (ppDrop !== true) {
-					extraPP += ppDrop || 0;
+				if (!sourceEffect || sourceEffect.id === 'pursuit') {
+					let extraPP = 0;
+					for (let i = 0; i < targets.length; i++) {
+						let ppDrop = this.singleEvent('DeductPP', targets[i].getAbility(), targets[i].abilityData, targets[i], pokemon, move);
+						if (ppDrop !== true) {
+							extraPP += ppDrop || 0;
+						}
+					}
+					if (extraPP > 0) {
+						pokemon.deductPP(move, extraPP);
+					}
 				}
-			}
-			if (extraPP > 0) {
-				pokemon.deductPP(move, extraPP);
-			}
-		}
-
-		if (!this.singleEvent('TryMove', move, null, pokemon, target, move)) {
-			move.mindBlownRecoil = false;
-			return false;
-		}
-
-		if (!this.runEvent('TryMove', pokemon, target, move)) {
-			move.mindBlownRecoil = false;
-			return false;
-		}
+				if (!this.singleEvent('TryMove', move, null, pokemon, target, move) || !(this.runEvent('TryMove', pokemon, target, move))) {
+					move.mindBlownRecoil = false;
+				} else {
+					this.singleEvent('UseMoveMessage', move, null, pokemon, target, move);
 
-		this.singleEvent('UseMoveMessage', move, null, pokemon, target, move);
+					if (move.ignoreImmunity === undefined) {
+						move.ignoreImmunity = (move.category === 'Status');
+					}
 
-		if (move.ignoreImmunity === undefined) {
-			move.ignoreImmunity = (move.category === 'Status');
-		}
+					if (move.selfdestruct === 'always') {
+						this.faint(pokemon, pokemon, move);
+					}
 
-		if (move.selfdestruct === 'always') {
-			this.faint(pokemon, pokemon, move);
-		}
+					let damage = false;
+					if (move.target === 'all' || move.target === 'foeSide' || move.target === 'allySide' || move.target === 'allyTeam') {
+						damage = this.tryMoveHit(target, pokemon, move);
+						if (damage || damage === 0 || damage === undefined) moveResult = true;
+					} else if (move.target === 'allAdjacent' || move.target === 'allAdjacentFoes') {
+						if (!targets.length) {
+							noTarget = true;
+						} else {
+							if (targets.length > 1) move.spreadHit = true;
+							let hitTargets = [];
+							for (let i = 0; i < targets.length; i++) {
+								let hitResult = this.tryMoveHit(targets[i], pokemon, move);
+								if (hitResult || hitResult === 0 || hitResult === undefined) {
+									moveResult = true;
+									hitTargets.push(targets[i].toString().substr(0, 3));
+								}
+								if (damage !== false) {
+									damage += hitResult || 0;
+								} else {
+									damage = hitResult;
+								}
+							}
+							if (move.spreadHit) this.attrLastMove('[spread] ' + hitTargets.join(','));
+							if (move.mindBlownRecoil && !move.multihit) {
+								this.damage(Math.round(pokemon.maxhp / 2), pokemon, pokemon, null, true);
+							}
```

Comment:
Didn't this block get moved to `moves.js`?

Reply:
Yes it was, thanks for catching that.